composer.json

```diff
@@ -10,18 +10,20 @@
     ],
     "require": {
         "php": "^5.6 || ^7.0",
-        "php-http/httplug": "^1.0",
+        "php-http/httplug": "^2.0",
```

Comment:
Do we want to force devs to install `php-http/httplug` to >=2.0? I do not see any significant code changes, so probably `"php-http/httplug": "^1.0 || ^2.0",` is what we need here?